0.6.x alpha3: Crash with SSL and own CA

I just tried to run confd and it crashes while initializing. I am new to `go`. Am I doing something wrong? If not, could you suggest a work-around?

```
$ cat /etc/confd/confd.toml
backend = "etcd"
client_cert = "/etc/etcd/pki/peer.crt"
client_key = "/etc/etcd/pki/peer.key"
client_cakeys = "/etc/etcd/pki/ca.crt"
confdir = "/etc/confd"
debug = true
interval = 600
noop = false
prefix = "/"
quiet = false
scheme = "https"
verbose = true
$
$ sudo ./bin/confd -scheme="https" -node <ec2-ip-redacted>:4001
2014-08-21T00:37:52Z my-server ./bin/confd[31795]: NOTICE Backend set to etcd
2014-08-21T00:37:52Z my-server ./bin/confd[31795]: NOTICE Starting confd
2014-08-21T00:37:52Z my-server ./bin/confd[31795]: NOTICE Backend nodes set to <ec2-ip-redacted>:4001
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x58 pc=0x555f4b]

goroutine 16 [running]:
runtime.panic(0x72b320, 0x942df3)
    /usr/local/go/src/pkg/runtime/panic.c:279 +0xf5
github.com/coreos/go-etcd/etcd.(*Client).internalSyncCluster(0x0, 0xc2080009d0, 0x1, 0x1, 0x6)
    /home/admin/confd-stuff/gopath/src/github.com/coreos/go-etcd/etcd/client.go:296 +0x15b
github.com/coreos/go-etcd/etcd.(*Client).SetCluster(0x0, 0xc2080009d0, 0x1, 0x1, 0x16)
    /home/admin/confd-stuff/gopath/src/github.com/coreos/go-etcd/etcd/client.go:279 +0x45
github.com/kelseyhightower/confd/backends/etcd.NewEtcdClient(0xc2080009d0, 0x1, 0x1, 0xc2080500e0, 0x16, 0xc2080504c0, 0x16, 0xc208050520, 0x14, 0x686740, ...)
    /home/admin/confd-stuff/gopath/src/github.com/kelseyhightower/confd/backends/etcd/client.go:30 +0x17d
github.com/kelseyhightower/confd/backends.New(0xc208000804, 0x4, 0xc208050520, 0x14, 0xc2080500e0, 0x16, 0xc2080504c0, 0x16, 0xc2080009d0, 0x1, ...)
    /home/admin/confd-stuff/gopath/src/github.com/kelseyhightower/confd/backends/client.go:31 +0x3e3
main.main()
    /home/admin/confd-stuff/gopath/src/github.com/kelseyhightower/confd/confd.go:29 +0x19b

goroutine 19 [finalizer wait]:
runtime.park(0x4155f0, 0x9478d0, 0x945b69)
    /usr/local/go/src/pkg/runtime/proc.c:1369 +0x89
runtime.parkunlock(0x9478d0, 0x945b69)
    /usr/local/go/src/pkg/runtime/proc.c:1385 +0x3b
runfinq()
    /usr/local/go/src/pkg/runtime/mgc0.c:2644 +0xcf
runtime.goexit()
    /usr/local/go/src/pkg/runtime/proc.c:1445

goroutine 20 [syscall]:
os/signal.loop()
    /usr/local/go/src/pkg/os/signal/signal_unix.go:21 +0x1e
created by os/signal.init·1
    /usr/local/go/src/pkg/os/signal/signal_unix.go:27 +0x32
```

I think my config above is correct, because I can access `etcd` with `curl`:

```
$ sudo curl --cacert /etc/etcd/pki/ca.crt --cert /etc/etcd/pki/peer.crt --key /etc/etcd/pki/peer.key -sL https://<ec2-ip-redacted>:4001/v2/stats/leader | python -m json.tool
{
    "followers": {
    <redacted>
    }, 
    "leader": <redacted>
}

```

--- a/backends/etcd/etcdutil/client.go
+++ b/backends/etcd/etcdutil/client.go
@@ -24,8 +24,9 @@ type EtcdClient interface {
 // It returns an error if a connection to the cluster cannot be made.
 func NewEtcdClient(machines []string, cert, key string, caCert string) (*Client, error) {
 	var c *etcd.Client
+	var err error
 	if cert != "" && key != "" {
-		c, err := etcd.NewTLSClient(machines, cert, key, caCert)
+		c, err = etcd.NewTLSClient(machines, cert, key, caCert)
 		if err != nil {
 			return &Client{c}, err
 		}

